### PR TITLE
NOISSUE - Add certs field

### PR DESCRIPTION
**Pull request by mteodor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ The service is configured using the environment variables presented in the follo
 | MF_EXPORT_MQTT_CLIENT_PK      | Client key for authentication in case when MTLS = true        | thing.key             |
 | MF_EXPORT_MQTT_QOS            | MQTT QOS                                                      | 0                     |
 | MF_EXPORT_MQTT_RETAIN         | MQTT retain                                                   | false                 |
-| MF_EXPORT_CONF_PATH           | Configuration file                                            | config.toml           |
+| MF_EXPORT_CONFIG_FILE           | Configuration file                                            | config.toml           |
```
Comment (blokovi) [inline README.md:127]: I think there are two extra white characters so table is not aligned

File: cmd/main.go [hunk 2]
```diff
@@ -175,9 +175,9 @@ func loadConfigs() (exp.Config, error) {
 			MTLS:       mqttMTLS,
 			SkipTLSVer: mqttSkipTLSVer,
 
-			CAPath:      mainflux.Env(envMqttCA, defMqttCA),
-			CertPath:    mainflux.Env(envMqttCert, defMqttCert),
-			PrivKeyPath: mainflux.Env(envMqttPrivKey, defMqttPrivKey),
+			CAPath:            mainflux.Env(envMqttCA, defMqttCA),
+			ClientCertPath:    mainflux.Env(envMqttCert, defMqttCert),
+			ClientPrivKeyPath: mainflux.Env(envMqttPrivKey, defMqttPrivKey),
```
Comment (dusanb94) [inline cmd/main.go:180]: Please consider renaming `ClientPrivKey`, but I have no better naming proposal. :)

File: .env [hunk 3]
```diff
@@ -4,7 +4,7 @@
 MF_NATS_URL=http://localhost:4222
 MF_EXPORT_PORT=8178
 MF_EXPORT_LOG_LEVEL=debug
-MF_EXPORT_CONF_PATH=/config.toml
+MF_EXPORT_CONFIG_FILE=/config.toml
```
Comment (drasko) [inline .env:7]: Is this `/` necessary?

File: README.md (hunk 1 above)
Comment (drasko) [reply README.md:127]: Looks like default value has a `/` prefix, and this one does not.

**Comment by mteodor** [comment]
This PR is included in #22 , closing it

**Pull request closed without merging**